core/common/src/main/java/alluxio/Constants.java

```diff
@@ -160,6 +161,12 @@
   public static final String UNDERFS_S3_ADMIN_THREADS_MAX = "alluxio.underfs.s3.admin.threads.max";
   public static final String UNDERFS_S3_UPLOAD_THREADS_MAX =
       "alluxio.underfs.s3.upload.threads.max";
+  public static final String UNDERFS_S3A_SECURE_HTTP_ENABLED =
+      "alluxio.underfs.s3a.secure.http.enabled";
+  public static final String UNDERFS_S3A_SERVER_SIDE_ENCRYPTION_ENABLED =
```

Comment:
Ideally it would be `s3n` and `s3a`, but for backward compatibility I did not modify the `s3` parameters.